apps/server/src/modules/account/services/account.service.spec.ts

```diff
@@ -404,6 +405,34 @@ describe('AccountService', () => {
 		});
 	});
 
+	describe('updatePassword', () => {
+		it('should update password but no other information', async () => {
+			const mockTeacherAccountDto = AccountEntityToDtoMapper.mapToDto(mockTeacherAccount);
+			const newPassword = 'newPassword';
+			const ret = await accountService.updatePassword(mockTeacherAccount.id, newPassword);
+
+			expect(ret).toBeDefined();
+			expect(ret).toMatchObject({
+				...mockTeacherAccountDto,
+				password: bcrypt.hash(newPassword, 10),
```

Comment:
The hash generated by bcrypt is not necessarily the same. Better check with bcrypt.compare(password, passwordHash)

Reply:
Checked ok. using compare instead of re-hashing